tests/foreman/api/test_template_combination.py

```diff
@@ -77,3 +78,19 @@ def test_positive_get_combination(self):
         self.assertEqual(self.template.id, combination.config_template.id)
         self.assertEqual(self.env.id, combination.environment.id)
         self.assertEqual(self.hostgroup.id, combination.hostgroup.id)
+        combination.delete()
```

Comment:
I had to revert it back. On sat 6.3 the get test was failing because they added a consistency constrante to not allow template removal with pending combination.